crates/uv-toolchain/src/downloads.rs

```diff
@@ -0,0 +1,52 @@
+#[derive(Debug, PartialEq)]
+pub struct PythonVersionDownload {
+    name: ImplementationName,
+    arch: Arch,
+    os: Os,
+    libc: Libc,
+    major: i64,
+    minor: i64,
+    patch: i64,
+    url: &'static str,
+    sha256: Option<&'static str>,
+}
```

Comment:
Not sure what we'll need here. Just using the same representation we have in the JSON for now.